main.go

```diff
@@ -372,3 +392,45 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func handleTriggerUninstall() {
+	triggerUninstallFlagSet := pflag.NewFlagSet("trigger-uninstall", pflag.ExitOnError)
+
+	var deploymentName, deploymentNamespace, policyNamespace string
+
+	triggerUninstallFlagSet.StringVar(
+		&deploymentName, "deployment-name", "config-policy-controller", "The name of the controller Deployment object",
+	)
+	triggerUninstallFlagSet.StringVar(
+		&deploymentNamespace,
+		"deployment-namespace",
+		"open-cluster-management-agent-addon",
+		"The namespace of the controller Deployment object",
+	)
+	triggerUninstallFlagSet.StringVar(
+		&policyNamespace, "policy-namespace", "", "The namespace of where ConfigurationPolicy objects are stored",
+	)
+	triggerUninstallFlagSet.AddGoFlagSet(flag.CommandLine)
+
+	_ = triggerUninstallFlagSet.Parse(os.Args[2:])
+
+	if deploymentName == "" || deploymentNamespace == "" || policyNamespace == "" {
+		fmt.Fprintln(os.Stderr, "--deployment-name, --deployment-namespace, --policy-namespace must all have values")
+		os.Exit(1)
+	}
+
+	terminatingCtx := ctrl.SetupSignalHandler()
```

Comment:
I'm wondering if this should have a maximum time before we cancel it.

Reply:
Sure, I'll set it to default to 5 minutes.